bin/regenerate.php

```diff
@@ -65,11 +95,28 @@ function add_simple_schemas() {
 			case 'type':
 				$key = 'post-types';
 				break;
+			
+			case 'wp_block':
+				$key = 'blocks';
+				$title = 'Editor Block';
+				$plural = 'Editor Blocks';
+				break;
+			
+			case 'wp_block-revision':
+				$key = 'block-revisions';
+				$title = 'Block Revision';
```

Comment:
Should this be Editor Block Revision to match the main `wp_block` resource?

Reply:
I thought about that and I decided it felt overly complicated. Blocks are pretty widely understood, I just think on the block page itself the additional clarity was helpful. Can make the change if you think it's worthwhile though